wgpu-types/src/lib.rs

```diff
@@ -360,14 +360,14 @@ bitflags::bitflags! {
         /// Allows the user to create uniform arrays of textures in shaders:
         ///
         /// ex.
-        /// `var textures: binding_array<texture_2d<f32>, 10>` (WGSL)\
+        /// `var textures: binding_array<texture_2d<f32>, 10>` (WGSL)
```

Comment:
If we're removing the slash - let's add two spaces so that they show up on separate lines. Hopefully ijr doesn't have an issue with that.

Reply:
Yes, two spaces can also work properly.

Reply:
But rustfmt seems to have a problem with that.

Reply:
Bleh, let's just add a new paragraph then